### ISSUE TITLE
cassandra won't cluster because etcd plugin fails trying to delete a non-existent file

### ISSUE THREAD

**Issue by dtheobald** [Metaswitch/clearwater-cassandra#87]
#### Symptoms

When bring up a fresh system, cassandra was failing to cluster properly. I noticed this log in the etcd cluster manager's log file:
```
29-03-2016 16:11:52.852 UTC ERROR synchronization_fsm.py:67 (thread CassandraPlugin): Call to CassandraPlugin.on_joining_cluster with cluster {u'192.168.165.81': u'joining, acknowledged change'} caused exception OSError(2, 'No such file or directory')
```
I tracked this down to the cassandra_plugin trying to remove a file (```/etc/clearwater/force_cassandra_yaml_refresh```) that didn't exist.

SInce this was blocking homestead nodes from comming up, I've temporarily made the following change to ```cassandra_plugin.py```:
```
--- a/clearwater-cassandra/usr/share/clearwater/clearwater-cluster-manager/plugins/cassandra_plugin.py
+++ b/clearwater-cassandra/usr/share/clearwater/clearwater-cluster-manager/plugins/cassandra_plugin.py
@@ -173,7 +173,8 @@ class CassandraPlugin(SynchroniserPluginBase):
             run_command("chown -R cassandra /var/lib/cassandra")
 
         self.start_cassandra()
-        os.remove("/etc/clearwater/force_cassandra_yaml_refresh")
+        if os.path.exists("/etc/clearwater/force_cassandra_yaml_refresh"):
+            os.remove("/etc/clearwater/force_cassandra_yaml_refresh")
 
     def get_seeds(self, cluster_view):
         seeds_list = []
```
While I was looking at this problem, I noticed that the following line was referring to the wrong file:
```
run_command("killall $(cat /var/lib/cassandra/cassandra.pid)", log_error=False)
```
It should be:
```
run_command("killall $(cat /var/run/cassandra/cassandra.pid)", log_error=False)
```
#### Impact

Homestead nodes won't function properly.

#### Release and environment 

release-94

**Comment by JimBunch** [comment]
Can anybody explain why we are suddenly seeing this problem? Did we introduce a recent change that caused this problem?

**Comment by rkd-msw** [comment]
Yes, this was introduced by the fix to https://github.com/Metaswitch/clearwater-cassandra/issues/16.

**Comment by rkd-msw** [comment]
Fixed in dev by #94.
